src/DrWatson.jl

```diff
@@ -64,7 +65,7 @@ function __init__()
     @require DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0" begin
         include("result_collection.jl")
     end
-    if display_update
+    if env_var in keys(ENV) ? parse(Bool, ENV[env_var]) : display_update
```

Comment:
I think here we should define a proper block:
```julia
if env_var in keys(ENV)
    x = try 
        parse(Bool, ENV[env_var]) 
    catch
        display_update
    end
else 
    display_update
```
and use that output for safety

Reply:
Are OK w/ me just putting this logic in the `const display_update = true` line, i.e.

```julia
const display_update = if evn_ver in ...
   ...
else
   true
end

Reply:
Are OK w/ me just putting this logic in the `const display_update = true` line, i.e.

```julia
const display_update = if evn_ver in ...
   ...
else
   true
end